paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -293,6 +311,9 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
         )
     }
 
+    /*
+     * Add comments inside function on how `PaymentMethodMetadata` is being created
+     */
```

Comment:
should this be a constructor for PaymentMethodMetadata instead of being in the PaymentSheetLoader?